lava_test_plans/testcases/master/template-mmtests.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set KERNEL_BRANCH = KERNEL_BRANCH|default("") %}
+{% set OS_INFO = OS_INFO|default("") %}
+
+{% set test_name = "mmtests-" + config_file|replace('configs/config-', '') %}
```

Comment:
Is this more clear than 
{% set test_name = config_file|replace('configs/config-', 'mmtests-') %}

What do you think?